package.json

```diff
@@ -11,51 +24,33 @@
     "**"
   ],
   "sideEffects": false,
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/drcmda/react-three-fiber.git"
+  },
   "scripts": {
     "prebuild": "rimraf dist && npm run typegen",
     "build": "rollup -c && yarn copy",
     "prepare": "npm run build",
-    "eslint": "eslint src/**/*.{js,ts,jsx,tsx}",
+    "eslint": "eslint --fix {src,examples/src}/**/*.{js,ts,jsx,tsx}",
     "test": "jest",
     "typecheck": "tsc --noEmit --emitDeclarationOnly false --strict --jsx react",
     "typegen": "tsc && mv dist/src/* dist && rm -rf dist/src || true",
     "typegen:components": "npx ts-node src/components/generateExports.ts",
     "copy": "copyfiles package.json readme.md LICENSE dist && json -I -f dist/package.json -e \"this.private=false; this.devDependencies=undefined; this.optionalDependencies=undefined; this.scripts=undefined; this.husky=undefined; this.prettier=undefined; this.jest=undefined; this['lint-staged']=undefined;\"",
     "pack-dist": "cd dist && yarn pack && mv *.tgz .."
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "pretty-quick --staged --pattern \"**/*.*(js|jsx|ts|tsx)\""
```

Comment:
`pretty-quick` is great for getting started, but it doesn't actually run `eslint`. It only does formatting. I set up `lint-staged` to correctly lint + format any staged changes.